test/unit/core/sagas/products.ts

```diff
@@ -463,71 +463,19 @@ suite('products saga', ({ sinon, expect, spy, stub }) => {
         const receiveProductsAction: any = { c: 'd' };
         const biases = ['bias'];
         const request = { e: 'f', biasing: { biases }};
-        const searchRequest = { e: 'f' };
+        const searchRequest = <any>{ e: 'f' };
```

Comment:
It might be preferable for us to write it like this instead just for style, like with `flux` a few lines down, but it doesn't really matter.
```js
const searchRequest: any = { e: 'f' };
```